site/docs/pages/config/types.mdx

```diff
@@ -32,6 +32,7 @@ type OnchainKitConfig = {
   rpcUrl: string | null; // RPC URL for onchain requests. Defaults to using CDP Node if the API Key is set
   chain: Chain; // Chain must be provided as we need to know which chain to use
   schemaId: EASSchemaUid | null; // SchemaId is optional as not all apps need to use EAS
+  walletCapabilities: WalletCapabilities; // Capabilities of the wallet - see EIP-5792
```

Comment:
Thank you for closing this!

We don't want to document something we are going to have a breaking changes soon.